packages/validation/src/user-guards.ts

```diff
@@ -1,5 +1,16 @@
-import { UserResponse, ExternalAuthorResponse } from '@asap-hub/model';
+import {
+  UserResponse,
+  ExternalAuthorResponse,
+  EventSpeakerUser,
+} from '@asap-hub/model';
 
 export const isInternalUser = (
   author: ExternalAuthorResponse | UserResponse,
 ): author is UserResponse => (author as UserResponse).email !== undefined;
+
+export const isInternalEventUser = (
+  user: EventSpeakerUser | undefined,
```

Comment:
Urgh, this is a weird type. I would have expected two types one InternalEventSpeaker one ExternalSpeaker.

Reply:
Actually i think External speakers are coming later